os/drivers/lcd/image_lib.c

```diff
@@ -0,0 +1,213 @@
+/****************************************************************************
```

Comment:
you can discuss with team internally.
IMO, this is not only related to LCD. Other module can use this in future..